lib/Runtime/Debug/TTSnapObjects.cpp

```diff
@@ -779,12 +789,15 @@ namespace TTD
             writer->WriteBool(NSTokens::Key::boolVal, snapFuncInfo->HasSuperReference, NSTokens::Separator::CommaSeparator);
         }
 
-        void ParseAddtlInfo_SnapScriptFunctionInfo(SnapObject* snpObject, FileReader* reader, SlabAllocator& alloc)
-        {
```

Comment:
It looks like some braces/whitespace was changed here (and elsewhere). We need to get it back into the correct ChakraCore style.

Reply:
should be fixed now.